Applications/WebPortal/WebApp/library/source/components/button.tsx

```diff
@@ -56,7 +56,7 @@ export class Button extends React.Component<Properties> {
       outline: 'none',
       MozAppearance: 'none' as 'none',
       cursor: 'pointer' as 'pointer'
-    },
```

Comment:
One of the reasons for casting to ```as React.CSSProperties``` is to remove many of the ```'none' as 'none'```, ```'pointer' as 'pointer'```, and instead write simply ```'none'```, ```'pointer'```, so go ahead and remove the extraneous casts.